components/table/index.css

```diff
@@ -33,8 +35,10 @@
 	--spectrum-table-drop-zone-background-color: rgb(var(--spectrum-drop-zone-background-color-rgb), var(--spectrum-drop-zone-background-color-opacity));
 	--spectrum-table-drop-zone-outline-color: var(--spectrum-accent-visual-color);
 
-	/* Row Selection */
-	--spectrum-table-row-active-color: rgb(var(--spectrum-gray-900-rgb), var(--spectrum-table-row-down-opacity)); /* active/down state background color */
+	/* Row States */
+	/* @todo Refactor or remove these properties once the RGB stripped value is available. */
+	--spectrum-table-row-background-color-hover: rgb(var(--spectrum-gray-900-rgb), var(--spectrum-table-row-hover-opacity));
+	--spectrum-table-row-background-color-active: rgb(var(--spectrum-gray-900-rgb), var(--spectrum-table-row-down-opacity)); /* active/down state background color */
 
 	/* Border */
 	--spectrum-table-border-color: var(--spectrum-gray-300);
```

Comment:
@rise-erpelding Just wanted to let you know I made that card for refactoring the table `border-block-start`s to `border-block-end`s that match Figma & React. 

CSS-1244
Our original discussion: https://github.com/adobe/spectrum-css/pull/3799#discussion_r2127796546